### DISCUSSION TITLE
What is Maizzle rendering engine?

### DISCUSSION THREAD

**Discussion by montogeek** [maizzle/framework#627] [Q&A] [answered]
Hi!

I am curious about that is Maizzle using to render the actual HTML.

Also, it is possible to use a different one?

Thanks!

**Comment by montogeek** [comment, marked as answer]
I see that it is using `posthtml`, and that there is no way to change it.

**Reply by cossssmin** [reply]
Yup, Maizzle uses PostHTML for template rendering. There are currently no plans to change that.
